friends/src/App.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import axios from "axios";
+import ReactDOM from "react-dom";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: []
+    };
+  }
+
+  componentDidMount() {
```

Comment:
Well done with getting the information from the server, via the GET and setting state. The instructions state that the logic of rendering the friends however, should be in a separate component! Perhaps one called FriendsList? Remember the fundamentals of React is breaking things into components! 😉